packages/scrollable-video/src/components/video.js

```diff
@@ -208,8 +208,6 @@ const ForwardRefVideo = React.forwardRef(
     }, [pickedSourceSrc, setVideoError, setVideoLoading, preloadCacheType, ref])
     return (
       <Video
-        playsInline
```

Comment:
It seems to me that `playsInline` attribute is not for autoplay but [for iOS not to enter fullscreen mode and continue to play inline](https://developer.apple.com/documentation/webkit/delivering_video_content_for_safari#3030250).

From this perspective, I suppose we still need `playsInline` as an attribute for video ?

Reply:
Oops, I will resolve this one since `playsInline` is added back in next patch.